modules/gltf/doc_classes/GLTFDocument.xml

```diff
@@ -115,9 +115,16 @@
 		</method>
 	</methods>
 	<members>
+		<member name="fallback_image_format" type="String" setter="set_fallback_image_format" getter="get_fallback_image_format" default="&quot;None&quot;">
+			The user-friendly name of the fallback image format. This is used when exporting the glTF file, including writing to a file and writing to a byte array.
+			This property may only be one of "None", "PNG", or "JPEG", and is only used when the [member image_format] is not one of "None", "PNG", or "JPEG". If it is desired to have multiple extension image formats, that can be done within a [GLTFDocumentExtension] class - this property only covers the use case of providing a base glTF fallback image when using a custom image format.
+		</member>
+		<member name="fallback_image_quality" type="float" setter="set_fallback_image_quality" getter="get_fallback_image_quality" default="0.25">
+			The quality of the fallback image, if any. For PNG files, this downscales the image on both dimensions by this factor. For JPEG files, this is the lossy quality of the image. This value is recommended to be low, since including multiple high quality images in a glTF file defeats the file size gains of using a more efficient image format.
+		</member>
 		<member name="image_format" type="String" setter="set_image_format" getter="get_image_format" default="&quot;PNG&quot;">
 			The user-friendly name of the export image format. This is used when exporting the glTF file, including writing to a file and writing to a byte array.
-			By default, Godot allows the following options: "None", "PNG", "JPEG", "Lossless WebP", and "Lossy WebP". Support for more image formats can be added in [GLTFDocumentExtension] classes.
+			By default, Godot allows the following options: "None", "PNG", "JPEG", "Lossless WebP", and "Lossy WebP". Support for more image formats can be added in [GLTFDocumentExtension] classes. A single extension class may provide multiple options for the specific format to use, or even an option that uses multiple formats at once.
```

Comment:
```suggestion
			By default, Godot allows the following options: "None", "PNG", "JPEG", "Lossless WebP", and "Lossy WebP". Support for more image formats can be added in [GLTFDocumentExtension] classes. A single extension class can provide multiple options for the specific format to use, or even an option that uses multiple formats at once.
```
Avoid "may" as it is easily confusing, assuming here the intended meaning is "is possible", which is also considered incorrect

Reply:
I assume the intent here is mentioning **the ability** of extensions to provide multiple options.

"May" -> _"has the possibility of"_
"Can" -> _"Is able to"_

Many contributors use "_may_", "_might_", "_can"_, "_could_", etc. interchangeably, but it would be nice if we catch it sooner than later, as well as provide this general guideline.

Reply:
I have been planning to add such a guideline as "may" is just too confusing, especially as "is possible" is non-standard and "technically" wrong

Reply:
Any future sweep or documentation aside "may" here should be changed IMO, even if it's used in the correct sense of "is permitted to", as it's too vague